translations/en.i18n.json

```diff
@@ -11,7 +11,7 @@
       "Spam / Commercials": "Spam / Commercials",
       "close": "Close",
       "report_placeholder": "Describe what you saw...",
-      "default_content_type": "Choose a reason for reporting this content",
+      "default_content_type": "Choose a reason for reporting this content (required)",
```

Comment:
plugin has 14 translates, I believe all of them have to get changees

Reply:
right, we are waiting for Sagi to get the translations for all languages. when we will get them the rest will be changed as well.